package.json

```diff
@@ -60,6 +60,7 @@
     "expect.js": "^0.3.1",
     "express": "^4.16.4",
     "express-session": "^1.16.2",
+    "http": "0.0.0",
```

Comment:
@nkpremices I'm curious as to why this is `0.0.0`. Can you send me a link to where you found this library?

Reply:
I didn't check the version @MemunaHaruna 
I'm also surprised. 
But I've checked [here](https://www.w3schools.com/nodejs/nodejs_http.asp) and saw that it is a built-in package. I shouldn't have installed it.

Reply:
Alright, you might want to uninstall it, then.